compiler/src/dmd/typesem.d

```diff
@@ -7479,6 +7479,62 @@ MATCH implicitConvToThroughAliasThis(TypeStruct from, Type to)
     return MATCH.nomatch;
 }
 
+/*******************************************
+ * Compute number of elements for a (possibly multidimensional) static array,
+ * or 1 for other types.
+ * Params:
+ *  t = static array type
+ *  loc = for error message
+ * Returns:
+ *  number of elements, uint.max on overflow
+ */
+final uint numberOfElems(Type t, const ref Loc loc)
```

Comment:
no point for this being `final`.

Reply:
Good point, and I think the style checker checks for this.